src/utils.rs

```diff
@@ -47,6 +47,35 @@ pub fn from_relocatable_to_indexes(relocatable: Relocatable) -> (usize, usize) {
         (relocatable.segment_index as usize, relocatable.offset)
     }
 }
+// ================
+//   RunResources
+// ================
+
+/// Maintains the resources of a cairo run. Can be used across multiple runners.
+#[allow(dead_code)]
+pub(crate) struct RunResources {
+    n_steps: Option<u64>,
```

Comment:
Maybe we could make this a `usize` to keep it in line with `ExecutionResources.n_steps`

Reply:
noted, fixed